packages/mdc-textfield/bottom-line/foundation.js

```diff
@@ -54,7 +54,7 @@ class MDCTextFieldBottomLineFoundation extends MDCFoundation {
   /**
    * @param {!MDCTextFieldBottomLineAdapter=} adapter
```

Comment:
We should probably remove the `=` since this param is no longer optional (ditto below)